modules/ROOT/pages/conf-openid-connect-task.adoc

```diff
@@ -1,19 +1,19 @@
-= Configure OpenID Connect Identity Management
+= Configuring OpenID Connect Identity Management
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-This task topic covers two identity management procedures for client registration in OpenID Connect:
+There are two identity management procedures for client registration in OpenID Connect:
 
 * Dynamic registration
 * Manual registration
 
-MuleSoft verifies support in Anypoint Platform for the following registrations:
+Anypoint Platform supports the following registrations:
 
 * Clients created dynamically in Okta and OpenAM
 * Clients created manually in Salesforce, Okta, OpenAM, and PingFederate
 
-Please note, that although integration with the aforementioned Identity providers have been officially tested, Anypoint platform supports the OpenID Connect Protocol. This means that, any Identity Provider that supports the protocol should be able to integrate unless they diverge from the http://openid.net/specs/openid-connect-core-1_0.html[specification].
+Although integration with the aforementioned Identity providers has been officially tested, Anypoint Platform supports the OpenID Connect Protocol. This means that any Identity Provider that supports the protocol can integrate unless they diverge from the http://openid.net/specs/openid-connect-core-1_0.html[specification].
```

Comment:
```suggestion
MuleSoft has officially tested the integration with these identity providers. Because Anypoint Platform supports the OpenID Connect Protocol, any identity provider that supports the protocol can integrate unless they diverge from the http://openid.net/specs/openid-connect-core-1_0.html[specification].
```